src/modules/commander/HealthAndArmingChecks/checks/estimatorCheck.cpp

```diff
@@ -534,17 +534,10 @@ void EstimatorChecks::checkEstimatorStatusFlags(const Context &context, Report &
 
 	if (_estimator_status_flags_sub.copy(&estimator_status_flags)) {
 
-		bool dead_reckoning = estimator_status_flags.cs_wind_dead_reckoning
-				      || estimator_status_flags.cs_inertial_dead_reckoning;
-
-		if (!dead_reckoning) {
-			// position requirements (update if not dead reckoning)
-			bool gps             = estimator_status_flags.cs_gps;
-			bool optical_flow    = estimator_status_flags.cs_opt_flow;
-			bool vision_position = estimator_status_flags.cs_ev_pos;
-
-			_position_reliant_on_optical_flow = !gps && optical_flow && !vision_position;
-		}
+		_position_reliant_on_velocity_aided_dead_reckoning = !estimator_status_flags.cs_inertial_dead_reckoning
```

Comment:
Does this even matter? Couldn't "relaxed local position validity" simply be vehicle_local_position.xy_valid?

Reply:
Good point, I'll check if there is any reason why it should be different